flixel/FlxObject.hx

```diff
@@ -548,16 +539,26 @@ class FlxObject extends FlxBasic
 	/**
 	 * Overriding this will force a specific color to be used for debug rect.
 	 */
-	public var debugBoundingBoxColor:Null<Int> = null;
-	
-	public var debugColorScheme(default, null):DebugColorScheme;
+	public var debugBoundingBoxColor(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorNotSolid(default, set):Null<Int> = null;
+	public var debugBoundingBoxColorPartial(default, set):Null<Int> = null;	
```

Comment:
I think these should be `FlxColor` instead of `Null<Int>`? And also be initialized with their default values right away?